### PR TITLE
feat: Implement _MM_GET_ROUNDING_MODE

### PR DESCRIPTION
**Pull request by marktwtn**

### CODE REVIEW

File: sse2neon.h [hunk 1]
```diff
@@ -6243,6 +6244,42 @@ typedef struct {
...
+    } r;
+
+#if defined(__aarch64__)
+    asm volatile("mrs %0, FPCR" : "=r"(r.value)); /* read */
+#else
+    asm volatile("vmrs %0, FPSCR" : "=r"(r.value)); /* read */
+#endif
+
+    if (r.field.bit22) {
+        if (r.field.bit23) {
```
Comment (jserv) [inline sse2neon.h:6269]: Use shorter form:
return (r.field.bit23) ? _MM_ROUND_TOWARD_ZERO : _MM_ROUND_UP;

Comment (marktwtn) [reply sse2neon.h]: Done.

**Comment by jserv** [comment]
@JishinMaster, can you help review the pull request?

**Comment by JishinMaster** [comment]
That seems fine by me.

**Comment by marktwtn** [comment]
@JishinMaster thank you for reviewing.

**Pull request merged**